src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -94,32 +108,32 @@ public OAuth2AccessToken readAccessToken(String token) {
     /**
      * Revokes the access tokens of the user with the given ID.
      *
-     * @param id
-     *            the user ID
-     * @param token
-     *            the token to access the service
+     * @param id    the user ID
+     * @param token the token to access the service
      */
     public void revokeAccessTokens(String id, String token) {
         AccessToken accessToken = new AccessToken.Builder(token).build();
-        OsiamConnector connector = createConnector();
-        connector.revokeAllAccessTokens(id, accessToken);
+        try {
+            connector.revokeAllAccessTokens(id, accessToken);
+        } catch (ConnectionInitializationException e) {
+            LOGGER.warn(String.format("Unable to revoke access token on auth-server %s: %s",
+                    authServerHome, e.getMessage()));
+            throw e;
+        }
```

Comment:
I know this is a little bit out of scope, but isn't is a good idea to catch [all possible exceptions thrown by the connector](https://github.com/osiam/connector4java/blob/f13e642222e1eafaaf61565504327b0f3e137998/src/main/java/org/osiam/client/AuthService.java#L322)? This would only be two more exceptions: `UnauthorizedException` and `ForbiddenException`. Because both should be generate `500 Internal Server Error` with an unspecific error message (right?) you can catch the common [`OsiamRequestException`](https://github.com/osiam/connector4java/blob/593c262d5e1a0ccf0735b58a914e582cc277a003/src/main/java/org/osiam/client/exception/OsiamRequestException.java).